OneCommunityGlobal/HighestGoodNetworkApp
# Raul implement opening a new tab by ctrl-clicking on the name in the user management

# Description
(PRIORITY LOW) Jae: Make Names Command(Mac)/Control(PC) Clickable :
a. Admin Login → Other Links → User Management
b. The clickable names in the Team Management page won’t allow me to Command(Mac)/Control(PC) click to open them in a new tab

## Mainly changes explained:
The old opening function was deleted and a new one was created. Now, besides the opening at the same tab by clicking, if the user clicks pressing the ctrl/command, the profile opens in a new tab.

## How to test:
1. git checkout raul_impl_opennewtab_ctrlclicking_name
2. npm install
3. npm run start:local
4. log as an admin user
5. go to Other Links -> User Management
6. click any name by pressing the ctrl and make sure that the name's profile will open in a new tab.

Here is a video of my testing below:
https://user-images.githubusercontent.com/29555732/213492966-f874773b-6b54-4876-a4e9-87d5774d9d79.mov
